src/components/challenge-rewards-drawer/ChallengeRewardsDrawer.tsx

```diff
@@ -285,13 +309,13 @@ export const ChallengeRewardsDrawer = ({
           <View
             style={[
               styles.statusCell,
-              isComplete ? styles.statusCellComplete : {}
+              hasCompleted ? styles.statusCellComplete : {}
             ]}
           >
             <Text
               style={[
                 styles.subheader,
-                isComplete ? styles.statusTextComplete : {},
+                hasCompleted ? styles.statusTextComplete : {},
```

Comment:
You can do `hasCompleted && styles.statusTextComplete`, but that kind of falls into the larger discussion around ternary vs boolean operators